src/implementations/vanilla/src/basics/grid/grid-item/grid-item.js

```diff
@@ -0,0 +1,35 @@
+const Template = require('./grid-item.html');
+var Interface = require('interface.json');
+var Core = require('_core.js');
+
+/**
+ * Creates a GridItem
+ *
+ * @class
+ */
+
+class GridItem extends Core {
+
+  constructor(options) {
+    super(options);
+    this._render(Template, options);
+  }
+
+  addSlot(slotElement){
+    this.mountPartialToComment('SLOT', slotElement);
+  }
+
+  changeFraction(fraction){
```

Comment:
I think `setFraction` would be more in line with how we've named similar functions.